server/app/views/applicant/ProgramIndexTemplate.html

```diff
@@ -29,7 +29,7 @@
 
 */-->
 <!doctype html>
-<html th:attr="lang=${preferredLanguage.code()}">
+<html th:attr="lang=${preferredLanguage.code()}" xmlns:cf="http://tempuri.org/">
```

Comment:
The xmlns is not necessarily required, but I think it will allow editors to not show the tags are warning/errors.

We'd want to replace tempuri.org with something relevant probably on civiform.us.